ISISComputingGroup/ibex_gui
# Ticket5798 refl opi performance

### Description of work
- Split each tab of Reflectometry OPI off into its own OPI so  GUI only needs to activate one at a time, improving performance when switching focus between scripting console and refl OPI.
- Full OPI still exists for use in device screens (e.g. on IDAaaS machines). This is easier than having to manually open 9 separate device screens, and since there is no console window in this perspective the focus switching is not an issue.
- Removed yellow highlighting of changed setpoints on alignment tab (redundant/misleading as parameters move immediately on pressing enter on this tab)
- Removed red highlighting for RBV not at setpoint on alignment tab (redundant)
- Fixed SP / SP:RBV / RBV column label alignment for groups on front panel for all instruments

#### To Test:
1. In your client, show reflectometry perspective by removing it from the list of hidden perspectives (l.2 in `base\uk.ac.stfc.isis.ibex.e4.client\plugin_customization.ini`)
1. Point your client at POLREF (or check out POLREF configs on your local machine)
1. Open reflectometry perspective
1. Check that all the tabs display correctly (i.e. parameter lists get populated)
1. Switch focus into console, then switch back to reflectometry OPI, verify it responds quickly e.g. enter a setpoint (compare to master - should be much faster with these changes)
1. Go to device screens
1. Check (only) full reflectometry OPI appears in device screen target selection, and displays the same content as the individual tabs

### Ticket

https://github.com/ISISComputingGroup/IBEX/issues/5798

### Acceptance criteria

- Reflectometry OPI is quick to respond when switching focus in reflectometry view (eg to genie_python console and back) 
- All individual OPIs display correctly
- Full OPI containing all tabs displays correctly (e.g. in device screens)

### Unit tests

/ 

### System tests

/

### Documentation

/

---

#### Code Review


### Final Steps
